src/components/Settings/Settings.tsx

```diff
@@ -132,6 +136,9 @@ export default function Settings(props: settingsProps) {
     // fetch request for every chapter in current book
   }
 
+  // function getUsfmSource(event) {
+  //   event.preventDefault()
+  // }
```

Comment:
Please remove dead code.  We can always get it back from the version history if needed.

Reply:
Yes